src/components/RelatedComponents.jsx

```diff
@@ -6,7 +6,7 @@ export default function RelatedComponents({ components }) {
       <h2 className="rel-component__heading">
         <img
           className="rel-component__heading--arrow"
-          src="/shapes/cmt_inventory_arrow.svg"
+          src="/images/cmt_inventory_arrow.svg"
```

Comment:
This change is out of scope, and the path is already up-to-date on `main`. Probably the result of rebasing.

Reply:
I have learned this the hard way! Thanks and I will always remember that! 🧠